tools/report-converter/codechecker_report_converter/report/parser/plist.py

```diff
@@ -519,6 +519,11 @@ def convert(
                         self._create_macro_expansion(
                             macro_expansion, file_index_map))
 
+            if report.annotations:
+                diagnostic["report-annotation"] = dict()
+                for key, value in report.annotations.items():
+                    diagnostic["report-annotation"][key] = value
```

Comment:
Why do we store the report annotations in the diagnostic here? I can't see it's usage.

Reply:
Because we parse a lot of plist files, merge its contents, and spit it back out. This means that we parse report-annotation info as well, and need it output it as well -- this last part was simply missing prior to this patch.